docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -0,0 +1,84 @@
+= Parameters
+
+The parent key for all of the following parameters is `cortex_tenant_ns_label`.
+
+== `app_name`
+
+[horizontal]
+type:: string
+default:: `cortex-tenant-ns-label`
+
+Configure the deployment name of the cortex-tenant-ns-label reverse proxy.
+
+== `config`
+
+[horizontal]
+type:: string
```

Comment:
As mentioned above, I'd make this `type:: dict`, so that we can actually take advantage of the hierarchical config.